Src/Test.UnitTests.Sarif.PatternMatcher.Cli/AnalyzeCommandTests.cs

```diff
@@ -17,6 +17,9 @@ namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Cli
 {
     public class AnalyzeCommandTests
     {
+        private const string SmallTargetName = "smallTarget.txt";
+        private const string LargeTargetName = "largeTarget.txt";
```

Comment:
do you really need this?

Reply:
Are you asking if we really need a different large target file? If so, yes. This allows us to confirm that when we are filtering out files that are too large to read, we are still able to process files that are under the threshold.

Reply:
I asked this because I didn't see you loading the file in memory. So, I was wondering if you need this or not.